kotlinx-nodejs/src/nonDeclarations.assert.assert.kt

```diff
@@ -0,0 +1,37 @@
+@file:Suppress("INTERFACE_WITH_SUPERCLASS", "OVERRIDING_FINAL_MEMBER", "RETURN_TYPE_MISMATCH_ON_OVERRIDE")
```

Comment:
I'd prefer if name of such files share some prefix with original files in the future

Reply:
sure! actually if you do believe that this too important to forget about we can regenerate now as well.

Reply:
Up to you. I don't think it's so importent.